alma/controllers/hook/StateHookController.php

```diff
@@ -132,6 +146,19 @@ public function run($params)
             default:
                 break;
         }
+
+        try {
+            $this->orderService->manageStatusUpdate($order, $newStatus);
+        } catch (\Exception $e) {
```

Comment:
manageStatusUpdate can trow other exception than AlmaException ?

Reply:
Yes :/ The deprecated RequestError from the PHP client